tests/report_test.go

```diff
@@ -87,6 +87,38 @@ func (c *ClientTests) TestReportTrackerCreate() {
 	assert.True(strings.HasPrefix(report.ID, "trkrep_"))
 }
 
+func (c *ClientTests) TestReportCustomColumnsCreate() {
+	client := c.TestClient()
+	assert := c.Assert()
+
+	report, _ := client.CreateReport(
+		"shipment",
+		&easypost.Report{
+			StartDate: c.fixture.ReportStartDate(),
+			EndDate:   c.fixture.ReportEndDate(),
+			Columns:   []string{"usps_zone"},
+		},
+	)
+
+	assert.Equal(reflect.TypeOf(&easypost.Report{}), reflect.TypeOf(report))
+}
+
+func (c *ClientTests) TestReportCustomAdditionalColumnsCreate() {
+	client := c.TestClient()
+	assert := c.Assert()
+
+	report, _ := client.CreateReport(
+		"shipment",
+		&easypost.Report{
+			StartDate:         c.fixture.ReportStartDate(),
+			EndDate:           c.fixture.ReportEndDate(),
+			AdditionalColumns: []string{"from_name", "from_company"},
+		},
+	)
+
```

Comment:
Add a comment here to explain why there is only one assertion, similar to what you have done on the other libs.

Reply:
Resolved in [ee78831](https://github.com/EasyPost/easypost-go/pull/63/commits/ee7883187e310394a3eabb3df65c7825e06effac)